engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/meta/GetShortTypeNameNode.java

```diff
@@ -0,0 +1,19 @@
+package org.enso.interpreter.node.expression.builtin.meta;
+
+import com.oracle.truffle.api.dsl.Specialization;
+import com.oracle.truffle.api.nodes.Node;
+import org.enso.interpreter.dsl.BuiltinMethod;
+import org.enso.interpreter.runtime.callable.atom.AtomConstructor;
+import org.enso.interpreter.runtime.data.Type;
+import org.enso.interpreter.runtime.data.text.Text;
+
+@BuiltinMethod(
+    type = "Meta",
+    name = "get_short_type_name",
+    description = "Gets the short name of a Type.",
+    autoRegister = false)
+public class GetShortTypeNameNode extends Node {
+  Text execute(Type type) {
+    return Text.create(type.getName());
```

Comment:
Internally it is called `type.getName()` and from `Meta` perspective it is `short_name`. Shouldn't it be `name` to keep some consistency?

I know `name` is already there and returns FQN - but maybe we can change it or have a default attribute `fqn=True`.

Reply:
I think we can change `short_name` to be `name` and current `name` to `qualified_name`. Does that sound ok?